sycl/include/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -9,7 +9,10 @@
 #pragma once
 
 #include <CL/sycl/access/access.hpp>
-#include <CL/sycl/detail/os_util.hpp> // for DLL_LOCAL used in int. header
+#include <CL/sycl/detail/defines.hpp>
```

Comment:
What is the reason to include defines.hpp header?

Reply:
`__SYCL_INLINE_NAMESPACE(cl)`